protocol/messenger_handler.go

```diff
@@ -687,7 +687,7 @@ func (m *Messenger) HandleRequestAddressForTransaction(messageState *ReceivedMes
 			Clock:       command.Clock,
 			Timestamp:   messageState.CurrentMessageState.WhisperTimestamp,
 			Text:        "Request address for transaction",
-			ChatId:      contactIDFromPublicKey(&m.identity.PublicKey),
+			ChatId:      messageState.CurrentMessageState.Message.GetChatId(),
```

Comment:
Looks good, I would only leave a comment on both that validation of `ChatId` is done in `handleCommandMessage`

Reply:
thanks, added the comment.